applications/sso/import-template/fdk-local-realm.template.json

```diff
@@ -919,7 +920,17 @@
     "contentSecurityPolicy" : "frame-src 'self'; frame-ancestors 'self'; object-src 'none';",
     "strictTransportSecurity" : "max-age=31536000; includeSubDomains"
   },
-  "smtpServer" : { },
+  "smtpServer" : {
+    "password" : "${SMTP_PASSWORD}",
+    "starttls" : "false",
+    "auth" : "true",
+    "port" : "465",
```

Comment:
Vil det alltid være port 465?

Reply:
som jeg forstår, det var to variant - 465 til SSL og 587 til STARTTLS 
https://www.fastmail.com/help/technical/ssltlsstarttls.html

gmail funket begge veis, så jeg bestemte å bruke bare en av de, til å redusere antall av variabler (2 mindre)